charts/oes/values.yaml

```diff
@@ -138,6 +138,12 @@ autopilot:
       keyStorePassword: dummypwd # Provide keystore password
       keyStoreType: PKCS12
       keyAlias: tomcat
+    server:
+      host:
+        dns:
+          name: ^https?://(?:localhost|OES_UI_LOADBALANCER_IP|opsmx.com)(?::[1-9]\d*)?/?
+    gate:
+      url: http://OES_GATE_IP:8084/
```

Comment:
Let's define OES-UI url and OES-Gate url at a single place and use them all over. Not define it under every section